{"ast":null,"code":"import{jsx as _jsx}from\"react/jsx-runtime\";import{jsxs as _jsxs}from\"react/jsx-runtime\";export var ErrorFallBack=function ErrorFallBack(_ref){var error=_ref.error,resetErrorBoundary=_ref.resetErrorBoundary;return/*#__PURE__*/_jsxs(\"div\",{role:\"alert\",children:[/*#__PURE__*/_jsx(\"p\",{children:\"Something went wrong:\"}),/*#__PURE__*/_jsx(\"pre\",{children:error.message}),/*#__PURE__*/_jsx(\"button\",{onClick:resetErrorBoundary,children:\"Try again\"})]});};","map":{"version":3,"names":["ErrorFallBack","_ref","error","resetErrorBoundary","_jsxs","role","children","_jsx","message","onClick"],"sources":["/Users/konami/Desktop/dev/udemy/discord-clone-udemy/src/utils/ErrorFallBack.tsx"],"sourcesContent":["import { ErrorBoundary } from \"react-error-boundary\";\nimport { FallbackProps } from \"react-error-boundary\";\n\nexport const ErrorFallBack = ({ error, resetErrorBoundary }: FallbackProps) => {\n  return (\n    <div role=\"alert\">\n      <p>Something went wrong:</p>\n      <pre>{error.message}</pre>\n      <button onClick={resetErrorBoundary}>Try again</button>\n    </div>\n  );\n};\n"],"mappings":"wFAGA,MAAO,IAAM,CAAAA,aAAa,CAAG,QAAhB,CAAAA,aAAaA,CAAAC,IAAA,CAAqD,IAA/C,CAAAC,KAAK,CAAAD,IAAA,CAALC,KAAK,CAAEC,kBAAkB,CAAAF,IAAA,CAAlBE,kBAAkB,CACvD,mBACEC,KAAA,QAAKC,IAAI,CAAC,OAAO,CAAAC,QAAA,eACfC,IAAA,MAAAD,QAAA,CAAG,uBAAqB,EAAI,cAC5BC,IAAA,QAAAD,QAAA,CAAMJ,KAAK,CAACM,OAAO,EAAO,cAC1BD,IAAA,WAAQE,OAAO,CAAEN,kBAAmB,CAAAG,QAAA,CAAC,WAAS,EAAS,GACnD,CAEV,CAAC"},"metadata":{},"sourceType":"module","externalDependencies":[]}